packages/sfpowerscripts-cli/src/commands/sfpowerscripts/orchestrator/publish.ts

```diff
@@ -321,14 +325,7 @@ export default class Promote extends SfpowerscriptsCommand {
     let cmd = `npm publish`;
 
     let tag: string;
-    if (this.flags.npmtag) {
-      tag = this.flags.npmtag;
-    } else if (packageMetadata.branch) {
-      tag = packageMetadata.branch
-    } else {
-      throw new Error(`Artifact ${packageName} ${packageVersionNumber} does not contain branch info. Please provide --npmtag flag explicitly, or re-build the artifact with the --branch flag.`);
-    }
-
+    tag = packageMetadata.branch
```

Comment:
Should guard against undefined/null tag

Reply:
is there a chance for branch to be null? as branch is a mandatory flag now